src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerSex = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${partnerSex}` : 'I am not married'}
```

Comment:
```suggestion
      <p className="Person__partner">
        {isMarried 
          ? `${partnerName} is my ${partnerSex}` 
          : 'I am not married'
        }
```

We can update formatting to improve code readability